CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Enhanced server logging to include load balancing and local worker info on startup.
 - Added 'Persistent' spatial class flag, typically used to override a non persistent base class.
 - Added a button to generate functional test maps from the editor. It can be found under **Window** > **Generate test maps**.
+- Added support for snapshot versioning, which enables 'out of date' snapshots to be flagged on servers and clients.
```

Comment:
How do I, an UnrealGDK customer with a game project, use this feature?

I'd suggest skimming [this](https://github.com/spatialos/UnrealGDK/blob/master/SpatialGDK/Extras/internal-documentation/how-to-write-good-release-notes.md#guiding-principles) and applying the WIIFM (what's in it for me?) here.